src/ec/packages/ec-component-site-header-core/site-header-core.html.twig

```diff
@@ -0,0 +1,190 @@
+{% spaceless %}
+
+{#
+  Parameters:
+    - "icon_file_path" (string) (default: ''): URL to icons file
+    - "logo" (associative array) (default: predefined structure): Logo image settings. format:
+        {
+          "title": (string) (default: ''): Logo title attribute.
+          "alt": (string) (default: ''): Logo alt attribute.
+          "href": (string) (default: ''): Logo URL.
+          "src": (string) (default: ''): Logo image file path, eg. dist/images/logo--en.svg.
+        },
+    - "language_selector" (associative array) (default: predefined structure): Language switcher settings. format:
+        {
+          "href": (string) (default: ''): URL for switcher
+          "name": (string) (default: ''): Switcher language label, eg. 'English', 'Français', etc.
+          "code": (string) (default: ''): Switcher language code, eg. 'en', 'fr', etc.
+          "overlay" (associative array) (default: predefined structure): Overlay language switcher settings. format:
+            {
+              "close_label": (string) (default: ''): Close button label eg. 'Close'.
+              "title": (string) (default: ''): Overlay title, eg. 'Select your language'.
+              "items": (array) (default: []): (array) (default: []): format: [
+                {
+                  "lang" (string) (default: '') Item language code, eg. 'en', 'fr', etc.
+                  "label" (string) (default: '') Item language label, eg. 'English', 'Français', etc.
+                  "path" (string) (default: '') Item language URL eg. '/example#language_en'.
+                  "active" (boolean) (default: false) define if item is the active language.
+                },
+              ],
+            },
+        },
+    - "search_form" (associative array) (default: predefined structure): EC Search Form component structure
+    - "auto_init" (boolean) (default: true)
+    - "extra_classes" (string) (default: '')
+    - "extra_attributes" (array) (default: []): format: [
+        {
+          "name" (string) (default: ''),
+          "value" (optional) (string)
+        ...
+      ]
+
+#}
+
+{# Default variables #}
+
+{% set _logo = logo|default({}) %}
+{% set _language_selector = language_selector|default({}) %}
+{% set _icon_file_path = icon_file_path|default('') %}
+{% set _auto_init = auto_init ?? true %}
```

Comment:
_auto_init parameter can be removed here, attribute is applied by default via the extra_attributes. To be checked in the site-header component also.

Reply:
oh yeah...even though i think we should give the chance not to auto-init something, why do we force it? would be better to have it by default and leaving the chance to suppress it.

Reply:
This was approved in previous PR (https://github.com/ec-europa/ecl-twig/pull/149#discussion_r326595029) with @emeryro, but maybe we can see with platform team if they need this option?